Sources/App/Github/GithubTableGenerator.swift

```diff
@@ -0,0 +1,25 @@
+public protocol Rowable {
+    
+    /// Should return all properties that will become table fields as a list of strings
+    ///
+    /// - Returns: table fields in the form of strings
+    func fields() -> [String]
+}
+
+public struct GithubTableGenerator {
```

Comment:
Big fan of this!

Reply:
Yeah this would be useful in a Markdown package